src/use_libretroshare.pri

```diff
@@ -62,18 +62,18 @@ rs_rnplib {
         # Windows msys2
         LIBRNP_LIBS *= -lbotan-3
     } else {
-        LIBRNP_LIBS *= -lbotan-2
+        # This is the case for macOS and other Unix-like systems
+        LIBRNP_LIBS *= -lbotan-3
```

Comment:
Will this works on older non-EOL macOS ? AFAIR we have some qmake functions to check for specific macOS versions if botan-2 is still needed for some of them